modules/core/src/test/java/org/apache/ignite/internal/managers/communication/GridIoManagerFileTransmissionSelfTest.java

```diff
@@ -840,6 +840,88 @@ public void testChunkHandlerCancelTransmission() throws Exception {
         }
     }
 
+    /**
+     * @throws Exception If fails.
+     */
+    @Test
+    public void testFileHandlerCrossConnections() throws Exception {
```

Comment:
This test does not need configured persistence region. This will significantly reduce the test execution time from ~5-10 seconds to 200 milliseconds.

Reply:
Fixed